src/main/io/serial.h

```diff
@@ -100,7 +100,7 @@ typedef enum {
     SERIAL_PORT_UART9,
     SERIAL_PORT_USART10,
     SERIAL_PORT_UART10 = SERIAL_PORT_USART10,
-
+    SERIAL_PORT_UART0,
```

Comment:
This will cause problems .. This value is for example used in CLI

Reply:
Correct, which is why I've tacked it on the end... not ideal at all.